packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/delete/delete.controller.js

```diff
@@ -0,0 +1,42 @@
+export default class AiDashboardRegistriesDeleteCtrl {
+  /* @ngInject */
+  constructor($translate, atInternet, AiDashboardService) {
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.AiDashboardService = AiDashboardService;
+  }
+
+  $onInit() {
+    this.loading = false;
+  }
+
+  deleteRegistryConfirm() {
+    this.atInternet.trackClick({
+      name:
+        'public-cloud::pci::projects::project::ai-dashboard::registries::delete::confirm',
+      type: 'action',
+    });
+
+    this.loading = true;
+
```

Comment:
add a finally block to reset  `this.loading` to `false`

Reply:
Fixed :)